index.js

```diff
@@ -29,7 +37,9 @@ export async function theme({Quasar, brand}) {
 
     // remove monkey-patch and apply variables
     CSSStyleDeclaration.prototype.setProperty = fn;
-    const cssVars = (await import('css-vars-ponyfill')).default;
+    const cssVars = (await import(
+      /* webpackChunkName: "CssVarsPonyfill" */
```

Comment:
Is this something special? When and why do you choose to add this statement for some imports and not others? What criteria do we need to use to do this consistently throughout our codebase? I saw that you were doing the same thing in this PR as well https://github.com/digitalbazaar/bedrock-vue/pull/4/files#diff-168726dbe96b3ce427e7fedce31bb0bcR22.

Reply:
+1 to using original name.